InscryptionAPI/Triggers/Interfaces.cs

```diff
@@ -1,240 +1,414 @@
-    int GetPassiveHealthBuff(PlayableCard target);
-}
\ No newline at end of file
+    public interface IGetOpposingSlots
+    {
+        /// <summary>
+        /// Indicates if this card wants to respond to getting the opposing slots
+        /// </summary>
+        /// <returns>True to provide opposing slots, False to leave the slots as default</returns>
+        bool RespondsToGetOpposingSlots();
+
+        /// <summary>
+        /// Gets the card slots that the card wants to attack
+        /// </summary>
+        /// <param name="originalSlots">The set of original card slots that the card would attack (as set by abilities in the base game)</param>
+        /// <param name="otherAddedSlots">Slots that have been added by other custom abilities</param>
+        /// <returns>The list of card slots you want to attack</returns>
+        /// <remarks>If your card is replacing the default attack slot (the opposing slot) see 'RemoveDefaultAttackSlot'
+        /// If you are **not** replacing the default attack slot, do **not** include that here. Simple ensure 'RemoveDefaultAttackSlot' returns false.
+        /// Only return the default attack slot if you want to attack it an additional time.</remarks>
+        List<CardSlot> GetOpposingSlots(List<CardSlot> originalSlots, List<CardSlot> otherAddedSlots);
+
+        /// <summary>
+        /// If true, this means that the attack slots provided by GetOpposingSlots should override the default attack slot. 
+        /// If false, it means they should be in addition to the default opposing slot.
+        /// </summary>
+        bool RemoveDefaultAttackSlot();
+    }
+    
+    /// <summary>
+     /// Data collection trigger that collects data related to attacked slots.
+     /// </summary>
+    public interface ISetupAttackSequence
+    {
+        /// <summary>
+        /// If true, this trigger will collect data from CollectModifyAttackSlots.
+        /// </summary>
+        /// <param name="card">Card whose slots will be modified.</param>
+        /// <param name="modType">Type of modification that is currently triggering.</param>
+        /// <param name="originalSlots">Original slots for attack.</param>
+        /// <param name="currentSlots">Current slots that are targeted.</param>
+        /// <param name="attackCount">How many attacks the card does.</param>
+        /// <param name="didRemoveDefaultSlot">True if the original (opposing) slot was removed by Bifurcated Strike or any other ability that does it.</param>
+        /// <returns>True if this trigger will collect data from CollectModifyAttackSlots.</returns>
+        public bool RespondsToModifyAttackSlots(PlayableCard card, OpposingSlotTriggerPriority modType, List<CardSlot> originalSlots, List<CardSlot> currentSlots, int attackCount, bool didRemoveDefaultSlot);
+        /// <summary>
+        /// Modifies data about targeted slots for attack.
```

Comment:
I think this needs to be a bit clearer. I know now that it modifies the opposing slots of any card, but the description is a bit confusing. Something more along the lines of `Modifies the slots that will be targeted of any card.`